graphql2/graphqlapp/messagelog.go

```diff
@@ -113,18 +117,44 @@ func (q *MessageLogConnectionStats) TimeSeries(ctx context.Context, opts *notifi
 		origin = *input.BucketOrigin
 	}
 
+	// validate segment type if given
+	// if input.SegmentBy != nil {
+	// 	var segments []string
+	// 	for _, segment := range graphql2.AllMessageLogSegmentBy {
+	// 		segments = append(segments, "\""+segment.String()+"\"")
+	// 	}
+
+	// 	if !slices.Contains(segments, input.SegmentBy.String()) {
+	// 		return nil, fmt.Errorf("invalid segment type. please use one of: %v", strings.Join(segments, ", "))
+	// 	}
+	// }
```

Comment:
still needed?

Reply:
This should still be needed, I couldn't decide if I wanted the validation in this function or the store function. I'll un-comment and it should be good to go with its import statement